alpa/api.py

```diff
@@ -231,3 +231,15 @@ def ret(*call_args, **call_kwargs):
         return mark_gradient((val, ggrad))
 
     return ret
+
+
+def grad_and_value(*args, **kwargs):
```

Comment:
This API seems redundant. You can just swap the return values in `alpa/torch/__init__.py::grad_and_value`

Reply:
Removed in favor of just using `value_and_grad`